lib/modules/manager/pipenv/artifacts.spec.ts

```diff
@@ -12,19 +12,26 @@ import {
 } from '../../../../test/util';
 import { GlobalConfig } from '../../../config/global';
 import type { RepoGlobalConfig } from '../../../config/types';
+import { logger } from '../../../logger';
 import * as docker from '../../../util/exec/docker';
+import type { ExtraEnv, Opt } from '../../../util/exec/types';
 import type { StatusResult } from '../../../util/git/types';
 import { find as _find } from '../../../util/host-rules';
-import { getPkgReleases as _getPkgReleases } from '../../datasource';
 import * as _datasource from '../../datasource';
+import { getPkgReleases as _getPkgReleases } from '../../datasource';
```

Comment:
this reorder is probably the cause of failing ci.

wonder why it's imported twice 🤔

Reply:
no clue, maybe intellij found it a great idea

Reply:
- #28337

Reply:
please merge main to fix this

Reply:
done :)